jep/15/README.adoc

```diff
@@ -0,0 +1,302 @@
+and to follow the requested allocation when possible.
+We may diverge from that in the case of emergency project needs.
+
+Decisions about allocating money are made at the link:https://www.jenkins.io/project/governance-meeting/[Jenkins Governance Meeting],
+unless there is an exception documented in another JEP
+(e.g. link:./8/README.adoc[JEP-8: Google Summer of Code Budgeting and Expenses]).
+Budget requests should be submitted to the developer mailing list and discussed there,
+the Governance meeting will be doing a formal approval/rejection based on the community feedback.
+
+==== Donations process
+
+Donations should be submitted through the CommunityBridge Funding system which supports donations by individuals and by organizations.
+It is possible to do a donation for "all project needs" or a targeted donation to any destination listed above.
+The process is documented link:https://docs.linuxfoundation.org/docs/communitybridge/communitybridge-funding/[here].
+
+For the Jenkins project all payment options are enabled:
+
+* One-time or monthly donations through via credit or debit card
+* One-time donations through invoices - for organizations making one-time donations
+
+==== Reimbursement process
+
+* Expenses should be pre-approved through the Jenkins developer mailing list and
+the link:https://www.jenkins.io/project/governance-meeting/[Governance meeting] **BEFORE** the expense happens.
+* The requester is registered on the CommunityBridge portal as a beneficiary
+* After the expense happens, the requester should submit an expense report through the CommunityBridge Expensify portal.
+* Expense reports should be approved by a CommunityBridge administrator
+
+=== Implementation
+
+==== CommunityBridge portal
+
+* CommunityBridge configuration should be updated according to this JEP
+* At least 3 Jenkins Governance Board members should be added as CommunityBridge administrators
+
+==== jenkins.io
+
+* link:https://www.jenkins.io/donate/[jenkins.io/donate] should be updated
+** _Why donate?_ should be updated to reflect motivation of this JEP and to reference non-infrastructure donations.
+** The page should reference this Jenkins Enhancement proposal.
+** Donations through SPI and ffis.de should be removed prom the page ads obsolete.
+
+==== GitHub Sponsor buttons
+
+One of the ways to facilitate the donations is to make donation buttons explicit on GitHub.
+There is a way to link:https://help.github.com/en/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository[displayi a sponsor button in a repository]
+which allows adding multiple links via a `FUNDING.yml` metadata file.
+
+* link:https://github.com/jenkinsci/[jenkinsci] GitHub organization -
+  Sponsor button is added to the Jenkins core and other core components.
+** Maintainers of plugins, tools, and subprojects can add the Sponsor buttons if they wish.
+   They can use the Jenkins project funding or to setup their own funding.
```

Comment:
Do we need to have a guidance document as to what is allowed for plugin developers to set up their own funding? We don't want to have cases where plugin developers have funding terms in opposition to the ideals of the project.

Reply:
Might be. Not sure it is a part of this JEP though. If you have ideas what to include, I am happy to add it to the JEP

Reply:
No ideas currently, just maybe add a note that sponsorship guidelines may be developed for plugins, etc?